### DISCUSSION TITLE
How to configure post logout redirect uris via the Admin REST API?

### DISCUSSION THREAD

**Discussion by dsteegen** [keycloak/keycloak#19087] [Q&A] [answered]
Hi,

We are currently in the process of migrating our Keycloak setup from version 18.0.0 to version 19.0.3. In our setup, we have a small sidecar container that creates clients when deployed on a fresh environment. After the migration, I noticed that the logout functionality is not behaving correctly due to the missing post logout redirect uri that has become mandatory since version 19.0.0.
When I manually update them with value '+' (as suggested in multiple other threads) it works fine. However, when checking the docs (https://www.keycloak.org/docs-api/19.0.3/rest-api/index.html#_clientrepresentation) I did not find an option to configure it via the API's.
Am I missing something? 

Thank you for helping me out or pointing me in the right direction ;)

**Comment by dsteegen** [comment, marked as answer]
I figured it out myself by inspecting the network call when saving a client via the admin UI. I turns out that the post logout redirect uris are sent via the attributes map in the request body:

```
"attributes": {
    "post.logout.redirect.uris": "+"
},
```

I will close this ticket 🙂

**Reply by Johniliam** [reply]
Ahhhhh finally thanksss, i discovered this way that to put more than one the use ## (there is no docs about it)
"post.logout.redirect.uris": "https://microsoft/*##https://google/*"
